README.md

```diff
@@ -104,9 +104,10 @@ Reference
 - [Azure Expiring Reserved Instances](./cost/azure/reserved_instances/expiration)
 - [Azure Reserved Instance Utilization](./cost/azure/reserved_instances/utilization/)
 - [Azure Reserved Instance Recommendations](./cost/azure/reserved_instances/mca_recommendations)
-- [Azure Unused IP Addresses](./cost/azure/unused_ip_addresses)
```

Comment:
Was this supposed to be removed with this change?

Reply:
Moved it to Line 110 to be in alphabetical order